packages/backend-modules/discussions/graphql/resolvers/_mutations/submitComment.js

```diff
@@ -28,8 +28,6 @@ module.exports = async (_, args, context) => {
     throw new Error(t('api/discussion/404'))
   }
 
-  Roles.ensureUserIsInRoles(me, discussion.allowedRoles)
-
```

Comment:
how come we are not checking for membership here?

Reply:
Line 42 already has     userCanComment(discussion, null, context), which checks that. This is redundant.

Reply:
good good